tests/unit/amo/components/TestCategory.js

```diff
@@ -436,6 +438,92 @@ describe(__filename, () => {
     });
   });
 
+  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    _categoriesFetch();
+    _categoriesLoad();
+    _getLanding();
+    _loadLanding();
+
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+
+    const root = render(
+      { _config: fakeConfig },
+      { autoDispatchCategories: false },
+    );
+
+    const landingShelves = root.find(LandingAddonsCard);
+
+    expect(landingShelves.at(0)).toHaveProp('footerLink', {
```

Comment:
It looks like this test is only necessary to cover the `addonType` logic, right? If so, you could simplify it like this:

```diff
diff --git a/tests/unit/amo/components/TestCategory.js b/tests/unit/amo/components/TestCategory.js
index 1c92fc4de..b674d778f 100644
--- a/tests/unit/amo/components/TestCategory.js
+++ b/tests/unit/amo/components/TestCategory.js
@@ -453,14 +453,10 @@ describe(__filename, () => {
 
     const landingShelves = root.find(LandingAddonsCard);
 
-    expect(landingShelves.at(0)).toHaveProp('footerLink', {
-      pathname: `/search/`,
-      query: {
-        addonType: ADDON_TYPE_THEMES_FILTER,
-        category: fakeCategory.slug,
-        featured: true,
-      },
-    });
+    expect(landingShelves.at(0)).toHaveProp('footerLink');
+    expect(landingShelves.at(0).props().footerLink.query.addonType).toEqual(
+      ADDON_TYPE_THEMES_FILTER,
+    );
 
     expect(landingShelves.at(1)).toHaveProp('footerLink', {
       pathname: '/search/',
```
It's not necessary to test the other parts of `query` because [this test](https://github.com/rebmullin/addons-frontend/blob/9080651ffe3f3bac00975457c6fb23552daf65b1/tests/unit/amo/components/TestCategory.js#L374) already checks them. It's good to remove all unnecessary assertions from a test because it will make failures easier to track down (there will be fewer failing tests).

The other tests can also be simplified in this same way.

Reply:
ok i see... i will update : )